kinova_driver/src/kinova_arm.cpp

```diff
@@ -254,48 +254,48 @@ KinovaArm::KinovaArm(KinovaComm& arm, const ros::NodeHandle& nodeHandle, const s
   home_trajectory_point.Position.Type = ANGULAR_POSITION;
   home_trajectory_point.Position.Delay = 0.0;
 
-  if (node_handle_.getParam("/ovis/arm/number_of_degree_per_sec", number_of_degree_per_sec) == false)
```

Comment:
Does this topic now appear directly on ` /number_of_degree_per_sec`? That's kind of hard to understand what it refers to when you look at the system as a whole.

Reply:
No it gets the param from the current namespace. So it's basically the relative path of the param, which is what we want in most cases. Specifying the absolute path can cause issues when we changing the namespace name for example. That's why these had to change in this case, because '/ovis/arm/...' no longer existed.

Reply:
Also it's not a topic just a param in the launch file